### DISCUSSION TITLE
Show / Hide Nav Bar

### DISCUSSION THREAD

**Discussion by MisterArbazzz** [h2oai/wave#2328] [Q&A]
Is there a way to show and hide navigation bar. 
I've been meaning to use the functionality to show or hide the navigation bar. is it possible?

Here is the image of what i am working with
<img width="233" alt="image" src="https://github.com/h2oai/wave/assets/87564754/0360322a-e3c1-47d4-9943-abf5f5e48665">

**Comment by marek-mihok** [comment]
@MisterArbazzz are you using the `ui.nav_card`? Sharing a minimal code example would be helpful.

In general, for hideable navigation you can use the header `nav` parameter. It will display a hamburger menu icon in the top left corner for toggling the navigation panel. https://wave.h2o.ai/docs/widgets/content/header/#with-navigation
